foundry/cameraPopoutAppWrapper.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Abstraction of the Application interface to be used with the Draggable class as a substitute for the app
+ * This class will represent one popout feed window and handle its positioning and draggability
+ */
+declare class CameraPopoutAppWrapper {
+
+  /**
+   * @param view- The CameraViews application that this popout belongs to
+   * @param userId- ID of the user this popout belongs to
+   * @param element- The div element of this specific popout window
+   */
+  constructor(view: CameraViews, userId: string, element: JQuery);
+
+  /**
+   * The CameraViews application that this popout belongs to
+   */
+  view: CameraViews;
+
+  /**
+   * The div element of this specific popout window
+   */
+  element: JQuery;
+
+  /**
+   * ID of the user this popout belongs to
+   */
+  userId: string;
+
+  /**
+   * Get the current position of this popout window
+   */
+  get position(): Application.Position;
+
+  /**
+   * Sets the position of this popout window
+   */
+  setPosition(position?: { left?: number; top?: number; width?: number; height?: number; scale?: number }): void;
```

Comment:
What I meant is that there should not be a parameter `position` but a parameter `{ left, top, width, height, scale }`. You can make the type `Partial<Application.Position>`

Reply:
Ah thanks, another thing learned for typing. Still new to this lol

Reply:
don't worry, we are all learning :)